Pages/AddNewSkill.cs

```diff
@@ -0,0 +1,146 @@
+﻿using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Collections.Specialized.BitVector32;
+using NUnit.Framework;
+
+namespace SpecProj2.Pages
+{
+    public class AddNewSkill
```

Comment:
you should create three pages Login Page, Skill Page and Language Page

Reply:
Please implement the POM Pattern, please see the below 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
same comments apply to all pages